Makefile

```diff
@@ -47,6 +47,10 @@ validate-docs:
 	python newsfragments/validate_files.py
 	towncrier --draft
 
+validate-and-test-docs: validate-docs
+	$(MAKE) -C docs doctest
```

Comment:
I think instead of this new job, I'd recommend using `build-docs` from within `validate-docs`. That also ensures that the *html*  output is also tested instead of only running the `doctest` job.